examples/lighting-app/esp32/sdkconfig.defaults

```diff
@@ -52,3 +52,6 @@ CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
 
 # Disable softap support by default
 CONFIG_ESP_WIFI_SOFTAP_SUPPORT=n
+# This example uses the older version of RMT driver to work with both
+# idf-v4.4.3 and idf-v5.0, so supressing the warnings by setting below option
+CONFIG_RMT_SUPPRESS_DEPRECATE_WARN=y
```

Comment:
Is this a temporary change?

Reply:
I'm guessing not, the newer rmt drivers are not present in v5.0. So better to keep this in examples rather than maintaining two different implementation under ifdefs.